services/app/src/reducers/Search.tsx

```diff
@@ -4,10 +4,12 @@ import {SearchChangeParamsAction, SearchResponseAction} from '../actions/ActionT
 import {getStorageString, setStorageKeyValue} from '../LocalStorage';
 import {SearchState} from './StateTypes';
 
+const LANGUAGE = 'language';
```

Comment:
maybe `LANGUAGE_KEY`? Otherwise it seems like the language itself is a constant.